dev-libs/hyprland-protocols/hyprland-protocols-0.2.ebuild

```diff
@@ -1,17 +1,23 @@
-# Copyright 2023 Gentoo Authors
+# Copyright 2023-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit meson
 
-COMMIT=4d29e48433270a2af06b8bc711ca1fe5109746cd
 DESCRIPTION="Wayland protocol extensions for Hyprland"
 HOMEPAGE="https://github.com/hyprwm/hyprland-protocols"
-SRC_URI="https://github.com/hyprwm/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
-S="${WORKDIR}/${PN}-${COMMIT}"
 
-KEYWORDS="amd64 ~riscv"
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/hyprwm/${PN}.git"
+	inherit git-r3
+else
+	COMMIT=4d29e48433270a2af06b8bc711ca1fe5109746cd
```

Comment:
When I first made this ebuild, there was no GitHub release for 0.2, hence why I used this commit. Now though the release exists for 0.2 (it was added several weeks later).
This PR would be a good opportunity to fix that and use the release instead.

Note: you will probably have to regenerate the manifest after doing this change.

Reply:
Done

Reply:
I just have to add -r1 to name of the file.
Since 0.2 was stabilize, I can drop 0.2 if you prefer